### PR TITLE
Aquatic Slime

### PR DESCRIPTION
**Pull request by Jackody2**
creation of a new slime called water slime

#### Summary
Content "Creation of a new type of slime"

#### Purpose of change

expand the slime family
#### Describe the solution

create a new slime
#### Describe alternatives you've considered

none
#### Testing

Applied and tested locally

### CODE REVIEW

**Comment by Daved27hundred** [comment]
May I suggest changing the name to something like "aquatic slime" or "buoyant slime" ?

**Comment by Jackody2** [comment]
yes it fits

File: data/json/monsters/slimes.json [hunk 1]
```diff
@@ -231,5 +231,39 @@
...
+    "morale": 100,
+    "melee_skill": 5,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "bleed_rate": 30,
+    "harvest": "exempt",
+    "special_attacks": [ [ "FORMBLOB", 30 ] ],
+    "death_function": {
+      "message": "The %s splits in two!",
```
[the remark this answers is not in the document]
Comment (Jackody2) [reply data/json/monsters/slimes.json:262]: right, then it won't split in two

**Comment by crypticcollaborator** [comment]
Is it intended that the aquatic blob is dramatically faster than the regular blob? 
If the aquatic blob has FORMBLOB and speed > 85, it can spawn additional small blobs over time, and these blobs won't be aquatic. That seems unintended.

**Comment by Jackody2** [comment]
for this reason I removed the ability to split after death, otherwise they would have generated normal slimes

**Comment by crypticcollaborator** [comment]
My understanding of the code is that formblob will make it generate slimes
before it does. It will also turn adjacent monsters into regular blobs as
well

**Comment by Jackody2** [comment]
fixed

File: data/json/monsters/slimes.json [hunk 2]
```diff
@@ -231,5 +231,33 @@
...
+    "aggression": 100,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 0 } ],
+    "bleed_rate": 30,
+    "harvest": "exempt",
+    "flags": [ "HEARS", "GOODHEARING", "NOHEAD", "POISON", "NO_BREATHE", "ACIDPROOF" ],
+    "armor": { "bash": 17, "cut": 9, "stab": 17, "acid": 12, "bullet": 7, "electric": 3 }
```
Comment (Fris0uman) [inline data/json/monsters/slimes.json:261]: should this have the same stats as a regular slime?

File: data/json/monstergroups/blob.json [hunk 3]
```diff
@@ -19,7 +19,8 @@
       { "monster": "mon_blob_small", "weight": 200, "cost_multiplier": 0 },
       { "monster": "mon_blob_small", "weight": 100, "cost_multiplier": 0, "pack_size": [ 2, 12 ] },
       { "monster": "mon_blob", "weight": 695, "cost_multiplier": 3, "pack_size": [ 5, 10 ] },
-      { "monster": "mon_blob_large", "weight": 5, "cost_multiplier": 6, "pack_size": [ 5, 10 ] }
+      { "monster": "mon_blob_large", "weight": 5, "cost_multiplier": 6, "pack_size": [ 5, 10 ] },
+      { "monster": "mon_aquatic_blob", "weight": 200, "cost_multiplier": 0, "pack_size": [ 5, 10 ] }
```
Comment (Fris0uman) [inline data/json/monstergroups/blob.json:23]: I don't see anything here to guarantee that they will spawn on water. As far as I can tell they'll just spawn among other slimes, so in slimes pits and labs pretty far from any pond.

**Pull request closed without merging**